bin/create-abell-app.js

```diff
@@ -74,6 +74,8 @@ program
     console.log(`\n${colors.green('>')} Finished Installing Dependencies 🤠`);
     console.log(`${colors.green('>')} Successfully Created ${projectName} 🚀`);
     console.log(`\n${colors.green('>>')} \`cd ${projectName}\` and \`${installCommand === 'yarn' ? 'yarn' : 'npm run'} dev\` to run dev-server ✨\n\n`);
+    // Exit
+    process.exit(-1);
```

Comment:
This should be 0

Negative exit code doesn't exit. This might bite us later.

Reply:
yeah I've fixed it. I was wondering why it worked for Piyush.